src/pattern-library/components/patterns/UtilitiesPage.js

```diff
@@ -70,6 +72,68 @@ export default function UtilitiesPage() {
             </button>
           </Library.Demo>
         </Library.Example>
+        <Library.Example title="Styling groups of inputs">
+          <p>
+            The <code>input-group</code> modifier allows adaptation of styling
+            on inputs when they are part of a group of inputs. This is identical
+            in functionality to the{' '}
+            <Link href="https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state">
```

Comment:
Per our a11y rules, this link should have a non-color indicator if its link-ness.